src/components/profile/PartyListGrid.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, Flex, Heading, Image, SimpleGrid, Text } from '@chakra-ui/react';
+
+const party_dummy_data = [
+  {
+    image: 'https://via.placeholder.com/140x140',
+    name: '가보자고',
+    id: '1',
+  },
+  {
+    image: 'https://via.placeholder.com/140x140',
+    name: '먹부림',
+    id: '2',
+  },
+  {
+    image: 'https://via.placeholder.com/140x140',
+    name: '겨울바다여행',
+    id: '3',
+  },
+  {
+    image: 'https://via.placeholder.com/140x140',
+    name: '퇴사기념',
+    id: '4',
+  },
+  {
+    image: 'https://via.placeholder.com/140x140',
+    name: '취업기념',
+    id: '5',
+  },
+  {
+    image: 'https://via.placeholder.com/140x140',
+    name: '확인1',
+    id: '6',
+  },
+  {
+    image: 'https://via.placeholder.com/140x140',
+    name: '확인2',
+    id: '7',
+  },
+];
+
+const PartyListGrid = () => {
+  return (
+    <Box mt='6' pb='10'>
+      <Heading size='md'>내 모임 목록</Heading>
+      <SimpleGrid mt='4' columns={3} spacing='10px'>
+        {party_dummy_data.map((party) => (
```

Comment:
p5;
```suggestion
        {party_dummy_data.map(({id, image, name}) => (
```
이렇게 하는거 어떤가용?

Reply:
오 좋습니다 바로 반영할게요!